pandasai/__init__.py

```diff
@@ -59,6 +62,18 @@ def __init__(
         self._is_conversational_answer = conversational
         self._verbose = verbose
         self._enforce_privacy = enforce_privacy
+        self._is_notebook = self.in_notebook()
+
+    def in_notebook(self):
```

Comment:
Maybe we can move it to a dedicate helper (like notebook or anything like that?

In this way we keep the code cleaner.

Reply:
> Sure @gventuri.
> 
> I could think of two approaches.
> 
> 1. Creating a folder named helpers inside pandasai and include notebook.py (we can add more helpers in the future)
> 2. a single python file named notebook.py inside pandasai.
> 
> Which one would you prefer?